ONSdigital/takeon-ui
# Feature/spp1271/new edit screen

# Description

Created 3 new templates to split functionality of the viewform into separate stages.

- View
- Edit
- Override

After survey selection user reaches the View Only screen where there are links to either the edit or override screens.
Once reaching one of these screens the user should be able to interact with the forms the same as previously but will only be able to edit data or override data depending on the view.
After the user has either chosen to save their changes or discard them, they are returned to the view only state again.

Functionality of form processing has been left the same with choice of templates being the only addition.
Created additional css file mulitpageCustom.css for styling of new UI pathway.






# How to test?
https://dc-ui-spp1271.es-take-on.aws.onsdigital.uk/

1. Choose any survey from general search, this should return with the view only page.
2. Choose either to edit or override the data via the links, this should change the view for that activity.
3. Complete the changes and either choose to save or discard them.
4. Return to the view only page and see the changes have occurred and the warnings/errors have updated accordingly.


Previous BDD tests should still work since functionality is identical.  (some changes might need to be made for navigation between features)


# Links

Links to Stories:

1.	SPP-1271 Implement new Edit screen design
    https://collaborate2.ons.gov.uk/jira/browse/SPP-1271

2.	SPP-1272 Implement new Override screen design
    https://collaborate2.ons.gov.uk/jira/browse/SPP-1272

Design brief used for new UI screens:
    https://miro.com/app/board/o9J_lUf0Gyw=/
